hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/RPC.java

```diff
@@ -901,7 +901,7 @@ public Builder setNumHandlers(int numHandlers) {
      * @return Default: -1.
      * @param numReaders input numReaders.
      */
-    public Builder setnumReaders(int numReaders) {
+    public Builder setNumReaders(int numReaders) {
```

Comment:
Is this ok from compatibility point of view, it is a public method in a public class. if any downstream project is using it, they will get compilation failures?
this method seems to be there since 2012, so pretty old as well...

Reply:
it'll break things. at best a new method can be added, internal code moved and this one tagged as deprecated

Reply:
Hi @tomscut @steveloughran @ayushtkn thanks help review it.

sorry for the late reply.

Considering that it is a public method in a public class and avoid compilation failures in downstream use, you can create a new method as @steveloughran  said, and keep the old method as deprecated, which may be better.

I will update PR later.